README.md

```diff
@@ -428,95 +428,20 @@ The Operator SDK CLI can generate CSV manifests via the following command:
 ```console
 $ operator-sdk olm-catalog gen-csv --csv-version 0.0.1
 ```
-For more details see the SDK's [CSV generation doc][csv_generation_doc].
-
-For the purpose of this guide, we will continue with this [predefined manifest][manifest_v1] file for the next steps. If you’d like, you can alter the image field within this manifest to reflect the image you built in previous steps, but it is unnecessary.
-
-### Deploy the Operator
 
-Deploying an operator is as simple as applying the operator’s CSV manifest to the desired namespace in the cluster.
+Then, after generate the CSV file will be required update it with the required information over the memcached-project which could not be done automatically by the tool. See an example of an [Memcached CSV][memcached_csv] file filled.
 
-First we need to create an [OperatorGroup][operator_group_doc] that specifies the namespaces that the operator will be targeting. Create the following OperatorGroup in the namespace where you will create the CSV. In this example the `default` namespace is used.
-
-```YAML
-apiVersion: operators.coreos.com/v1alpha2
-kind: OperatorGroup
-metadata:
-  name: memcached-operator-group
-  namespace: default
-spec:
-  targetNamespaces:
-  - default
-```
+**NOTE:** You are able to preview and validate your CSV file syntaxe in [OperatorHub.io CSV Preview][operator-hub-io-preview].
 
-Next create the CSV.
+### Testing locally
 
-```sh
-$ curl -Lo memcachedoperator.0.0.1.csv.yaml https://raw.githubusercontent.com/operator-framework/getting-started/master/memcachedoperator.0.0.1.csv.yaml
-$ kubectl apply -f memcachedoperator.0.0.1.csv.yaml
-$ kubectl get ClusterServiceVersion memcachedoperator.v0.0.1 -o json | jq '.status'
-```
-
-After applying this manifest, nothing has happened yet, because the cluster does not meet the requirements specified in our manifest. Create the CustomResourceDefinition and RBAC rules for the `Memcached` type managed by the operator:
-
-```sh
-$ kubectl create -f deploy/crds/cache_v1alpha1_memcached_crd.yaml
-$ kubectl create -f deploy/service_account.yaml
-$ kubectl create -f deploy/role.yaml
-$ kubectl create -f deploy/role_binding.yaml
-```
+The next step to check your project working with OLM will be generate its bundle. Following the [Testing your Operator with Operator Framework][testing-operators] to know how to perform it and test its integration.
```

Comment:
```suggestion
The next step is to ensure your project deploys correctly with OLM and runs as expected. Follow this [testing guide][testing-operators] to deploy and test your operator.
```

The bundle is created when you run `gen-csv --update-crds`. I suggest adding a note about that flag when `gen-csv` is run above.